charms/tensorboard-controller/src/charm.py

```diff
@@ -114,6 +114,7 @@ def service_environment(self) -> Dict[str, str]:
         gateway_ns, gateway_name = self._get_gateway_data()
         ret_env_vars = {
             "ISTIO_GATEWAY": f"{gateway_ns}/{gateway_name}",
+            "ISTIO_HOST": "*",
```

Comment:
Are we missing [this](https://github.com/kubeflow/manifests/blob/v1.9-branch/apps/tensorboard/tensorboard-controller/upstream/base/kustomization.yaml#L8) var as well?